Mathlib/CategoryTheory/Sites/Grothendieck.lean

```diff
@@ -306,7 +306,7 @@ instance : CompleteLattice (GrothendieckTopology C) :=
       · intro X S hS
         rw [trivial_covering] at hS
         apply covering_of_eq_top _ hS
-      · refine' @CompleteLattice.bot_le _ (completeLatticeOfInf _ isGLB_sInf) (trivial C))
+      · exact @CompleteLattice.bot_le _ (completeLatticeOfInf _ isGLB_sInf) (trivial C))
```

Comment:
Quite an interesting bug: `refine` or `exact` works but `refine'` enters an infinite loop ... I guess new use of `refine'` is discouraged now so probably there's no need to fix it ...

https://gist.github.com/alreadydone/5299197dc88b8237a4896803ac9a915f